src/Ssh/Ssh.AlcWrapper/AzureIdUtilities.cs

```diff
@@ -0,0 +1,83 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Azure.Commands.Ssh
+{
+    public static class AzureIdUtilities
```

Comment:
It seems like these utilities functions are intended for internal use only.
```suggestion
    internal static class AzureIdUtilities
```

Reply:
I removed this class, because I just found out that Azure PowerShell actually has a class for Parsing Azure ID